### PR TITLE
Execute functions instead of variables

### PR DESCRIPTION
**Pull request by szepeviktor**
The same goes for `SIMULATE` and `FORCE` and `VERBOSE`

---

This is a very bad PR.

All shell problems should be corrected.

### CODE REVIEW

File: scripts/drupal-backup-all [hunk 1]
```diff
@@ -146,7 +155,7 @@ while [ $# -gt 0 ] ; do
       # Test to see if we have site aliases defined for the specified site.
       # TODO: We assume 'live' is remote and 'backup' is local.  Confirm this?
       site=$option
-      if [ -z "$(drush site-alias @$site.live 2>/dev/null)" ] || [ -z "$(drush site-alias @$site.backup 2>/dev/null)" ] ; then
+      if [ -z "$(drush site-alias @${site}.live 2>/dev/null)" ] || [ -z "$(drush site-alias @${site}.backup 2>/dev/null)" ] ; then
```
Comment (szepeviktor) [inline scripts/drupal-backup-all:158]: Shell variables may contain spaces. So when referencing a variable:
```diff
- @$site.live
+ "@${site}.live"
```

Comment (szepeviktor) [reply scripts/drupal-backup-all:158]: If you put a human in the equation (meaning "we know that value will never contain a space"), that person should help running the script 24/7!

File: scripts/drupal-backup-all [hunk 2]
```diff
@@ -224,43 +233,43 @@ for site in ${backup_list} ; do
...
-        #echo drush -y -v rsync @$site.live:%private @$site.backup:%private
-        $EXEC drush -y -v rsync @$site.live:%private @$site.backup:%private
+        #echo drush -y -v rsync @${site}.live:%private @${site}.backup:%private
+        execute drush -y -v rsync @${site}.live:%private @${site}.backup:%private
       fi
     fi
 
-    drush @$site.backup sql-query --db-prefix 'select max(nid),max(changed) from {node}' > $backup_hash_file 2>/dev/null
+    drush @${site}.backup sql-query --db-prefix 'select max(nid),max(changed) from {node}' > $backup_hash_file 2>/dev/null
     if [ "x`diff $hash_file $backup_hash_file`" != "x" ] ; then
```
Comment (szepeviktor) [inline scripts/drupal-backup-all:273]:
```suggestion
    if [ -n "$(diff $hash_file $backup_hash_file)" ] ; then
```

File: scripts/drupal-backup-all [hunk 3]
```diff
@@ -302,36 +311,36 @@ for site in ${backup_list} ; do
...
         fi
 
         # Run the update
         update_list="${update_list} $site"
-        $SIMULATE && echo drush -y @$site.update pm-update 1>&2
-        $SIMULATE && echo drush -n @$site.update pm-updatecode > $update_log_file 2>&1
-        $SIMULATE || drush -y @$site.update pm-update > $update_log_file 2>&1
+        $SIMULATE && echo drush -y @${site}.update pm-update 1>&2
+        $SIMULATE && echo drush -n @${site}.update pm-updatecode > $update_log_file 2>&1
+        $SIMULATE || drush -y @${site}.update pm-update > $update_log_file 2>&1
```
Comment (szepeviktor) [inline scripts/drupal-backup-all:343]: Should we use `execute` here also?

File: scripts/drupal-backup-all (hunk 1 above)
Comment (greg-1-anderson) [reply scripts/drupal-backup-all:158]: Yes, quoting variables is always good practice even when you know it's unnecessary. Drush does not allow alias names to contain spaces or special characters, though, so in this case we really do know that this variable doesn't need to be quoted.

**Comment by greg-1-anderson** [comment]
I haven't used these scripts in over eight years, I just stuck them here for archival purposes when I was cleaning up.

**Pull request closed without merging**